data/sdo-howto-examples.txt

```diff
@@ -92,98 +92,342 @@ PRE-MARKUP:
-      <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-        <div itemprop="text">Remove the hubcap and loosen the lug nuts.</div>
-      </div>
-    </div>
-    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToStep">
-      <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-        <span itemprop="duringMedia"><img itemprop="image" href="replacing-tire.mp4" /></span>
-        <div itemprop="text">Remove the flat tire and put the spare tire on the exposed lug bolts.</div>
-      </div>
-    </div>
-    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToStep">
-      <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-        <div itemprop="text">Tighten the lug nuts by hand.</div>
-      </div>
-      <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToTip">
-        <div itemprop="text">Don't use the wrench just yet.</div>
-      </div>
-    </div>
-  </div>
-  <div itemprop="step" itemscope itemtype="http://schema.org/HowToSection">
-    <div itemprop="name">Finishing up</div>
-    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToStep">
-      <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-        <div itemprop="text">Lower the jack and tighten the lug nuts with the wrench.</div>
-      </div>
-    </div>
-    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToStep">
-      <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-        <div itemprop="text">Replace the hubcap.</div>
-      </div>
-    </div>
-    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToStep">
-      <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-        <div itemprop="text">Put the equipment and the flat tire away.</div>
-      </div>
+    <strong><span itemprop="name">Change a Flat Tire</span></strong>
+    <div>About <span itemprop="estimatedCost" itemscope itemtype="http://schema.org/MonetaryAmount">
+        <meta itemprop="currency" content="USD"></meta>
+        <meta itemprop="value" content="20"></meta>
+    </span>$20</div>
+    <div>About <span itemprop="totalTime" content="PT30M">30 minutes</span></div>
+    <div>Necessary Items:</div>
+    <div itemprop="tool" itemscope itemtype="http://schema.org/HowToTool">Spare tire</div>
+    <div itemprop="tool" itemscope itemtype="http://schema.org/HowToTool">Lug wrench</div>
+    <div itemprop="tool" itemscope itemtype="http://schema.org/HowToTool">Jack</div>
+    <div itemprop="tool" itemscope itemtype="http://schema.org/HowToTool">Wheel wedges</div>
+    <div itemprop="supply" itemscope itemtype="http://schema.org/HowToSupply">Flares</div>
+    <div itemprop="steps" itemscope itemtype="http://schema.org/ItemList">
```

Comment:
The field name is now called "step"

Reply:
Also "step" can be CreativeWork, HowToSection, HowToStep, Text. So I don't think ItemList is valid is it?

If the ItemList wrapper is removed then the sections can be steps like so:

```<div itemprop="step" itemscope itemtype="http://schema.org/HowToSection">```